andela/asgard-cfh
# Generate Jwt On Sucessful Signup/Signin

#### What does this PR do?
- Generate JWT for user authentication on successful signup/signin

#### Description of Task to be completed?
- jwt should be generated for authentication as oppossed to using sessions

#### How should this be manually tested?
-  visit application url [here](https://asgard-cfh-staging.herokuapp.com/)
- signup manually
- check the developers console,
- navigate to application and should see your token stored in the local storage

#### Any background context you want to provide?
- 
#### What are the relevant pivotal tracker stories?
[157801292](https://www.pivotaltracker.com/story/show/157801292)
[157801291](https://www.pivotaltracker.com/story/show/157801291)

#### Screenshots (if appropriate)
N/A

#### Questions:
- none
